centaur/test_cromwell.sh

```diff
@@ -36,7 +36,7 @@ RUN_DIR=$(pwd)
 TEST_THREAD_COUNT=16
```

Comment:
I'm assuming you shellchecked all of this :P

Reply:
I only shellcheck other peoples PRs. 😉